static/js/pages/App.js

```diff
@@ -355,7 +361,8 @@ const mapStateToProps = state => {
     snackbar,
     banner,
     showUserMenu,
-    profile: getOwnProfile(state)
+    audioPlayerLoaded: isAudioPlayerLoaded(state),
```

Comment:
you should be able to just use your `isAudioPlayerLoadedSelector` here (and in the `Drawer.js` too)